index.bs

```diff
@@ -460,23 +470,42 @@ agent MUST run the following steps:
    initialized to the result of [=DatagramDuplexStream/creating=] a {{DatagramDuplexStream}},
    its [=DatagramDuplexStream/create/readable=] set to |transport|.{{[[IncomingDatagrams]]}},
    and its [=DatagramDuplexStream/create/writable=] set to |transport|.{{[[OutgoingDatagrams]]}}.
-1. If the scheme of |parsedURL| is `https`, [=in parallel=],
-   [=initialize WebTransport over HTTP=].
+1. Let |transport| have a
+   <dfn attribute for="WebTransport">\[[Connection]]</dfn> internal slot initialized to null.
```

Comment:
It seems bad to define internal slots inside an algorithm... They should be defined under the object they belong to.

Reply:
+1. Filed #222.